src/tequila/circuit/compiler.py

```diff
@@ -462,7 +478,7 @@ def compile_controlled_rotation(gate: RotationGateImpl, angles: list = None) ->
 @compiler
 def compile_to_cc(gate) -> QCircuit:
```

Comment:
If I get it right, then the name should probably be adapted. Maybe "compile_to_single_control" (compile_to_c could be misinterpreted)

Reply:
with pycharm (I think you are using it?) you can to right-click and then "refactor -> rename" then it changes the name throughout the whole package. The compiler flags need to be adapted manually though.